File: aoj/ITP1/3/3_C.cpp
#include<iostream>
#include<algorithm>
#include<vector>
#include<map>

using namespace std;

int main(){
    vector<int> x,y;
    int tmpX = 1,tmpY = 1;

    while(true){
        cin >> tmpX >> tmpY;
        if(tmpX == 0 && tmpY == 0)break;

        x.push_back(tmpX);
        y.push_back(tmpY);
    }

    for(int i = 0; i < x.size(); i++){
        if(x[i] < y[i])
            cout << x[i] << " " << y[i] << endl;
        else 
            cout << y[i] << " " << x[i] << endl;
    
    }
    

    return 0;
}

File: aoj/golang/ITP1/03/3_C.go
package main

import (
    "fmt"
)

func main() {
    var x, y    []int
    var tmpX, tmpY int = 1, 1

    for {
        fmt.Scanf("%d %d", &tmpX, &tmpY)

        if tmpX == 0 && tmpY == 0 {
            break
        }

        x = append(x, tmpX)
        y = append(y, tmpY)
    }

    for i := 0; i < len(x); i++ {
        if (x[i] < y[i]) {
            fmt.Println(x[i], y[i])
        } else {
            fmt.Println(y[i], x[i])
        }
    }
}